src/core_plugins/kibana/public/home/components/home.js

```diff
@@ -149,33 +150,31 @@ export function Home({ addBasePath, directories }) {
           <EuiTitle size="l">
             <h1>Add Data to Kibana</h1>
           </EuiTitle>
+          <EuiText>
+            <p>
+              These turnkey solutions will help you quickly add data into Kibana and turn it into
+              pre-built dashboards and monitoring systems.
+            </p>
+          </EuiText>
         </EuiFlexItem>
 
         <EuiFlexItem grow={false}>
-          <EuiFlexGroup alignItems="center">
-            <EuiFlexItem grow={false}>
-              <p className="kuiText kuiSubduedText">
+          <EuiTextColor color="subdued">
+            <EuiText>
+              <p>
                 Data already in Elasticsearch?
               </p>
-            </EuiFlexItem>
-
-            <EuiFlexItem grow={false}>
-              <KuiLinkButton
-                buttonType="secondary"
-                href={addBasePath('/app/kibana#/management/kibana/index')}
-              >
-                Set up index patterns
-              </KuiLinkButton>
-            </EuiFlexItem>
-          </EuiFlexGroup>
-
+            </EuiText>
+          </EuiTextColor>
+          <EuiSpacer size="s" />
+          <a href="/app/kibana#/management/kibana/index" className="euiButton euiButton--primary euiButton--small">
```

Comment:
`EuiButton` doesn't have an href ability yet, so I'm manually building the HTML. We should replace this with a component later.